src/Symfony/Component/HttpClient/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Add IPv6 support to `NativeHttpClient`
  * Allow using HTTP/3 with the `CurlHttpClient`
+ * Added RFC 9111–based caching support to `CachingHttpClient` (though some features remain partially supported)
```

Comment:
```suggestion
 * Add RFC 9111–based caching support to `CachingHttpClient` (though some features remain partially supported)
```

Reply:
Adressed in https://github.com/symfony/symfony/pull/59576/commits/8a0b0340efdc9cae3357bd70c75f1448146cafeb